sdk/ml/azure-ai-ml/azure/ai/ml/entities/_component/parallel_component.py

```diff
@@ -237,11 +237,20 @@ def _attr_type_map(cls) -> dict:
 
     def _to_rest_object(self) -> ComponentVersionData:
         rest_object = super()._to_rest_object()
+        # schema required list while backend accept json string
         if self.partition_keys:
-            rest_object.properties.component_spec["partition_keys"]= \
+            rest_object.properties.component_spec["partition_keys"] = \
                 json.dumps(self.partition_keys)
         return rest_object
 
+    @classmethod
+    def _from_rest_object_to_init_params(cls, obj: ComponentVersionData) -> Dict:
+        init_kwargs = super()._from_rest_object_to_init_params(obj)
```

Comment:
@bupt-wenxiaole In case you feel interested, the logic will be moved here inside ParallelComponent after refactoring.